conjur/cli.py

```diff
@@ -37,6 +38,14 @@ def main_description():
         return '''Usage:
   conjur [global options] command [subcommand] [value] [--option=arg]'''
 
+    @staticmethod
+    def usage(*args):
```

Comment:
please remove and rebase with master. My PR was merged that has these changes so you don't need to add this by yourself